deploy/template.yaml

```diff
@@ -230,12 +224,17 @@ Resources:
               KBV_ENV_NAME: "/perfTest/kbvEnvName"
               CORE_STUB_USERNAME: "/perfTest/coreStubUsername"
               CORE_STUB_PASSWORD: "/perfTest/coreStubPassword"
+              CORE_URL: "/perfTest/coreUrl"
+              ORCH_STUB_URL: "/perfTest/orchStubUrl"
+              RP_STUB: "/perfTest/rpStub"
+              BASE_URL: "/perfTest/baseUrl"
               F2F_URL: "/perfTest/f2fUrl"
               K6_DYNATRACE_URL: "/perfTest/dynatraceUrl"
               HOME_URL: "/perfTest/homeUrl"
               SIGNIN_URL: "/perfTest/signInUrl"
               SMS_OTP: "/perfTest/fixedSMSOTP"
               EMAIL_OTP: "/perfTest/fixedEmailOTP"
+              PHONE_OTP: ""/perfTest/fixedPhoneOTP""
```

Comment:
SMS_OTP and PHONE_OTP are serving the same purpose. Please remove one of these and ensure only one of these env variable is used across the scripts.